### DISCUSSION TITLE
Heroku integration

### DISCUSSION THREAD

**Discussion by abernicchia-heroku** [PostgREST/postgrest#3819] [Q&A]
I helped with [updating that part on v11](https://postgrest.org/en/v11/integrations/heroku.html) but now I see it has dissapeared from v12 and the [repo](https://github.com/PostgREST/postgrest-heroku) is now archived (since Jan 2024).

I do assume it still works without any changes, so wondering why removing it.

Please, let me know if I can help with reinstating it.

Thanks

**Comment by wolfgangwalther** [comment]
It was removed in https://github.com/PostgREST/postgrest-docs/pull/735.
